gui/MainWindow.cxx

```diff
@@ -3572,7 +3572,11 @@ void MainWindow::computeCamera()
   auto landmarks = d->groundControlPointsHelper->registrationLandmarks();
 
   // Set up algorithm
-  auto config = kv::config_block::empty_config();
+  auto config = readConfig("resection_camera.conf");
+  if (!config)
+  {
+    config = kv::config_block::empty_config();
+  }
   config->set_value("algorithm:type", "ocv");
```

Comment:
I suppose this is more of an issue with the existing code, but it seems odd to call this algorithm "algorithm".  it would be better to call it "resection" or something more informative.  That algorithm name appears both in code and config files.  If we named every algorithm "algorithm" there would be name clashes and we wouldn't know what each algorithm was for if they get merged into a common config file.

Reply:
ML, thanks for the comments; changes commited: add gui_ prefix to resection_camera.conf for consistency, use resection name for algorithm

Reply:
now calling it "resection" instead of "algorithm"